addons/l10n_mx/migrations/2.1/post-migrate.py

```diff
@@ -0,0 +1,48 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import api, SUPERUSER_ID
+
+from odoo.addons.account.models.chart_template import update_taxes_from_templates
+
+
+def migrate(cr, version):
+    update_taxes_from_templates(cr, 'l10n_mx.mx_coa')
+
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    debit_tag = env.ref("l10n_mx.tag_debit_balance_account")
+    credit_tag = env.ref("l10n_mx.tag_credit_balance_account")
+    mx_companies = env['res.company'].search([('country_id', '=', env.ref('base.mx').id)])
+
+    # First update current entries
+    cr.execute("""
```

Comment:
I don't really understand why you are doing all this in SQL, it feels too complicated for such a tiny change :thinking: 

What I would do:
- Either you go in pre-migrate and do it in SQL - but you can do something way more simple I think - 
- Either you stay in post-migrate and make use of the ORM's availability.

```python
    env = api.Environment(cr, SUPERUSER_ID, {})
    debit_tag = env.ref("l10n_mx.tag_debit_balance_account")
    credit_tag = env.ref("l10n_mx.tag_credit_balance_account")
    account_102_ids = env['ir.model.data'].search([
        ('name', 'ilike', '%_cuenta102_02'),
        ('model', '=', 'account.account'),
    ]).mapped('res_id')
    accounts_102 = env['account.account'].search([('id', 'in', account_102_ids)])
    accounts_102.tag_ids = [Command.unlink(credit_tag.id), Command.link(debit_tag.id)]
```
I think something like this does the trick ?